src/openapi.yaml

```diff
@@ -6728,6 +6688,239 @@ components:
           type: string
           maxLength: 1000
           minLength: 1
+      discriminator:
+        propertyName: betrokkene_type
+    RolNatuurlijkPersoon:
+      title: betrokkene_identificatie
+      type: object
+      properties:
+        burgerservicenummer:
+          title: Burgerservicenummer
+          description: Het burgerservicenummer, bedoeld in artikel 1.1 van de Wet
+            algemene bepalingen burgerservicenummer.
+          type: string
+          maxLength: 9
+        nummerAnderNatuurlijkPersoon:
+          title: Nummer ander natuurlijk persoon
+          description: Het door de gemeente uitgegeven unieke nummer voor een ANDER
+            NATUURLIJK PERSOON
+          type: string
+          maxLength: 17
+        aNummer:
+          title: A nummer
+          type: integer
+          maximum: 2147483647
+          minimum: -2147483648
+          nullable: true
+        geslachtsnaam:
+          title: Geslachtsnaam
+          description: De stam van de geslachtsnaam.
+          type: string
+          maxLength: 200
+        voorvoegselGeslachtsnaam:
+          title: Voorvoegsel geslachtsnaam
+          type: string
+          maxLength: 80
+        voorletters:
+          title: Voorletters
+          description: De verzameling letters die gevormd wordt door de eerste letter
+            van alle in volgorde voorkomende voornamen.
+          type: string
+          maxLength: 20
+        voornamen:
+          title: Voornamen
+          description: Voornamen bij de naam die de persoon wenst te voeren.
+          type: string
+          maxLength: 200
+        geslachtsaanduiding:
+          title: Geslachtsaanduiding
+          description: Een aanduiding die aangeeft of de persoon een man of een vrouw
```

Comment:
I'm missing the inject mapping of value -> human readable presentation

Reply:
fixed